### PR TITLE
Docsite: improve reporting_bugs_and_features.rst

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Docsite: improve reporting_bugs_and_features.rst

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
`docs/docsite/rst/community/reporting_bugs_and_features.rst`

### CODE REVIEW

**Comment by Andersson007** [comment]
ready_for_review

File: docs/docsite/rst/community/reporting_bugs_and_features.rst [hunk 1]
```diff
@@ -54,3 +56,5 @@ Requesting a feature
 ====================
 
 The best way to get a feature into Ansible is to :ref:`submit a pull request <community_pull_requests>`, either against ansible-core or against a collection. See also :ref:`ansible_collection_merge_requirements`.
+
+You can also submit a feature request through opening an issue in the `ansible/ansible <https://github.com/ansible/ansible/issues>`_ for ``ansible-core`` or in a corresponding collection repository (refer to the :ref:`Bugs in collections<reporting_bugs_in_collections>` section to learn how to find a proper issue tracker).
```
Comment (Akasurde) [inline docs/docsite/rst/community/reporting_bugs_and_features.rst:60]:
```suggestion
You can also submit a feature request through opening an issue in the `ansible/ansible <https://github.com/ansible/ansible/issues>`_ for ``ansible-core`` or in a corresponding collection repository (refer to the :ref:`Bugs in collections<reporting_bugs_in_collections>` section to learn how to find a proper issue tracker).
```

**Comment by acozine** [comment]
Looks good, thanks for adding the anchor and the link @Andersson007.

**Comment by Andersson007** [comment]
@acozine thanks for reviewing and merging! I also created a backport to 2.11 https://github.com/ansible/ansible/pull/75450

**Pull request merged**